include/nbl/asset/utils/ISPIRVOptimizer.h

```diff
@@ -42,6 +42,7 @@ class ISPIRVOptimizer final : public core::IReferenceCounted
 
         core::smart_refctd_ptr<ICPUBuffer> optimize(const uint32_t* _spirv, uint32_t _dwordCount, system::logger_opt_ptr logger) const;
         core::smart_refctd_ptr<ICPUBuffer> optimize(const ICPUBuffer* _spirv, system::logger_opt_ptr logger) const;
+        const std::initializer_list<E_OPTIMIZER_PASS> getPasses() const;
```

Comment:
you still need to change the container for `m_passes`

Reply:
yes but then remember to return a `span<const E_OPTIMIZER_PASS>` for this method